### PR TITLE
OVS enum to free text change

### PR DESCRIPTION
**Pull request by palatsangeetha**

### CODE REVIEW

**Review by HenrikHL — COMMENTED** [review]
I would assume we would want to validate the content - or will  we only do that via a Database persistence error?

**Comment by douradofrazer** [comment]
> I would assume we would want to validate the content - or will we only do that via a Database persistence error?

The assumption here is that the data in the db has already been validated.
Since it's a read-only API, we do not need to have validation in place here.

File: ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/TransportCallTO.java [hunk 1]
```diff
@@ -13,7 +13,7 @@ public record TransportCallTO(
   String universalImportVoyageReference,
   String universalExportVoyageReference,
   PortTerminalLocation location,
-  PortCallStatusCode statusCode,
+  String statusCode,
```
Comment (douradofrazer) [inline ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/TransportCallTO.java:16]: Could you add a comment with a link to the possible values?
https://github.com/dcsaorg/DCSA-Information-Model/blob/master/datamodel/referencedata.d/portcallstatuscodes.csv

**Review by douradofrazer — CHANGES_REQUESTED** [review]

File: ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/TransportCallTO.java (hunk 1 above)
Comment (palatsangeetha) [reply ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/TransportCallTO.java:17]: Added the comment with the link and pushed the changes

**Pull request merged**